src/components/Button.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+  export let title: string = '';
+  export let action = () => {};
+</script>
+
+<div class="flex w-350 justify-between">
```

Comment:
should not need this styling. seems like leftover code. additionally, if you find this to be worth it, look into adding the svelte version of `{...props}` - a prop that can accept any standard HTML button prop.